pymhf/injected.py

```diff
@@ -35,7 +35,6 @@
     from pymhf.utils.config import canonicalize_setting
 
     log_level = _internal.CONFIG.get("logging", {}).get("log_level", "info")
-    log_level = "info"
```

Comment:
😅
The actual logging switch needs some work since I don't think it works properly right now, but this is good to fix anyway thanks!